lib/foreman_ansible/remote_execution.rb

```diff
@@ -36,6 +36,11 @@ def self.register_rex_feature
         :description => N_('Run an Ansible playbook against given hosts'),
         :provided_inputs => %w[playbook]
       )
+      RemoteExecutionFeature.register(
+        :ansible_run_capsule_upgrade,
+        N_('Upgrade capsules on given hosts'),
+        :description => N_('Upgrade capsules on given capsule server hosts')
```

Comment:
I think in both cases Capsules should be uppercase since we treat it as the proper name for a thing.